### ISSUE TITLE
Show suggestions from n-th character

### ISSUE THREAD

**Issue by dwelle** [atom/autocomplete#56]
It would be great if we could set the autocompleter to make suggestions only after you type n-th letters of a word. E.g. from 3rd character, `fu` wouldn't trigger autocomplete, while `fun` would.

This is useful when you have 1-2 letter snippets, and both autocomplete and snippets are triggered/ confirmed via `tab` key (I know you can map custom keys, but it's useful to use just one).

**Comment by kevinsawicki** [comment]
This package is deprecated and no longer shipped with Atom. If this is still an issue using the [autocomplete-plus](https://github.com/atom/autocomplete-plus) package, please open up a new issue on that repository, thanks.
